xbmc/interfaces/python/XBPython.cpp

```diff
@@ -45,7 +45,7 @@
 #include "interfaces/legacy/AddonUtils.h"
 #include "interfaces/python/AddonPythonInvoker.h"
 #include "interfaces/python/PythonInvoker.h"
-
+PyThreadState* savestate;
```

Comment:
Yeah, it was a temporary measure until a replacement to transfer variable from XVPython to Pythoninvoker since paxxi said it wouldn't be a good idea to use global variables.
@Rechi